File: test/csharp/SpecGate.CtscFixtures/Conformance/MultiFile/Greet.cs
using SpecGate.Annotations;

namespace SpecGate.CtscFixtures.Conformance.MultiFile;

public static class Greet
{
    [SpecOperation("greet", Spec = "fixture.multi_file")]
    public static string Execute([SpecInput("name")] string name) => $"Hello, {name}!";
}

File: test/rust/crates/specgate-ctsc-fixtures/src/conformance/multi_file/greet.rs
use specgate::spec_operation;

#[spec_operation("greet", spec = "fixture.multi_file")]
pub fn greet(name: &str) -> String {
    format!("Hello, {name}!")
}

#[test]
fn greets_from_first_source_file() {
    assert_eq!(greet("Ada"), "Hello, Ada!");
}
